titus-api/src/main/java/com/netflix/titus/api/connector/cloud/LoadBalancerConnector.java

```diff
@@ -37,5 +37,5 @@
      */
     Completable isValid(String loadBalancerId);
 
-    Single<Set<String>> getRegisteredIps(String loadBalancerId);
+    Single<LoadBalancer> getLoadBalancer(String loadBalancerId);
```

Comment:
Whenever we modify or create a new method with RX result, we should use Spring-Reactor, not RxJava.
If we do not, we will have to do it anyway in a few months from now.

Reply:
I have created a little script to estimate RxJava vs Spring-Reactor usage in our code base: https://github.com/Netflix/titus-control-plane/blob/master/rx.sh